packages/sign_in_with_apple/example/macos/Flutter/GeneratedPluginRegistrant.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  Generated file. Do not edit.
+//
+
+import FlutterMacOS
+import Foundation
+
+import sign_in_with_apple
+
+func RegisterGeneratedPlugins(registry: FlutterPluginRegistry) {
+    SignInWithApplePlugin.register(with: registry.registrar(forPlugin: "SignInWithApplePlugin"))
```

Comment:
In order to get this working on macOS at all, I had to manually edit this file (and then run via Xcode instead of Flutter, as the latter would just overwrite it again).

Furthermore I had to put the Obj-C / header bridge files in place, even though for example `url_launcher` is just plain Swift. Not sure if that works a little differently because it has a `macOS` specific package, or if something in these files still suggest Obj-C…

Flutter for macOS also needs a globally installed cocoapods for now to detect the plugin dependencies, but in this case that still doesn't work (even though a `bundle pod install` does add the dependency to the Xcode project).

cc @HenriBeck

Reply:
Fixed in e57894321cda7732a48a4b27dddcded58dd23849